torch/_torch_docs.py

```diff
@@ -4102,6 +4130,34 @@ def merge_dicts(*dicts):
     tensor([-0.1369, -1.1740, -0.6460, -0.1929])
 """.format(**single_dim_common))
 
+add_docstr(torch.minimum, r"""
+minimum(input, other, *, out=None) -> Tensor
+
+Compares the element-wise minimum of :attr:`input` and :attr:`other`.
+
+.. note::
+    If one of the elements being compared is a NaN, then that element is returned.
+    If both elements are NaNs then the first is returned.
+    The complex NaNs are defined as at least one of the real or imaginary parts being a NaN.
+
+Args:
+    {input}
+    other (Tensor): the second input tensor
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> torch.minimum(torch.tensor([[2.1, 1], [-float('inf'), 6]]), torch.tensor([[1.2, 2], [4, float('nan')]]))
+    tensor([[1.2000, 1.0000],
+            [  -inf,    nan]])
+    >>> torch.minimum(torch.tensor([complex(2, 1), complex(1, 2)]), torch.tensor([complex(1, 10), complex(1, 8)]))
```

Comment:
These examples may be removed.

Reply:
The examples of complex numbers are removed.